rskj-core/src/main/java/org/ethereum/crypto/signature/Secp256k1.java

```diff
@@ -92,4 +95,18 @@ static void reset() {
         instance = new Secp256k1ServiceBC();
         initialized = false;
     }
-}
\ No newline at end of file
+
+    private static void checkLoadError() {
+        Throwable loadError = null;
+
+        // Workaround; TODO: remove workaround when Secp256k1Context is fixed
+        try {
```

Comment:
Seem's like quite a big workaround 🤔 [Secp256k1Context](https://github.com/rsksmart/native/pull/21/files#diff-7153beecae9f3dd66af7743a3fbb11d503a05ba937aac465298b313534be7641R52) has `getLoadError`, doesn't it?

Reply:
yeah, probably it is. Removed it.